rake_nltk/rake.py

```diff
@@ -166,20 +166,12 @@ def _generate_phrases(self, sentences):
            @return: List of List of strings where each sublist is a collection
```

Comment:
Would you please change the @return to
`@return: List of string tuples where each tuple is a collection of words forming a contender phrase.`
so that code change and corresponding documentation are part of a single commit.